src/jace/api.py

```diff
@@ -72,18 +75,19 @@ def jit(
         fun: Function to wrap.
         primitive_translators: Use these primitive translators for the lowering to SDFG.
             If not specified the translators in the global registry are used.
-        kwargs: Jit arguments.
+        kwargs: Jit arguments, see `JITOptions` for more.
 
     Note:
         This function is the only valid way to obtain a JaCe computation.
     """
-    if kwargs:
+    not_supported_jit_keys = kwargs.keys() - {"backend"}
```

Comment:
Wouldn't it be better to use the information in the TypedDict?
```suggestion
    not_supported_jit_keys = kwargs.keys() - JITOptions.__annotations__.keys()
```

Reply:
Thanks. I was sure that this was possible bud did not know how to do.